### DISCUSSION TITLE
Use WinGet to install community PowerToys Run plugins

### DISCUSSION THREAD

**Discussion by hlaueriksson** [microsoft/winget-cli#3949] [Q&A]
Is it possible to install community [PowerToys Run](https://learn.microsoft.com/en-us/windows/powertoys/run) plugins with WinGet?

Community plugins are usually distributed as zip files that are manually extracted to `%LocalAppData%\Microsoft\PowerToys\PowerToys Run\Plugins`.

Can this be done with WinGet instead?

**Comment by Trenly** [comment]
Theoretically, yes, but the hard part would be setting the path to extract the Zip file to. This could be achieved by setting the flag to require an install location, but that still has the potential not to work for all installs depending on user settings. I would imagine that a community developed DevHome extension would be better for this application than winget would be

**Reply by hlaueriksson** [reply]
Thanks, I will take a look at DevHome.

**Reply by denelon** [reply]
https://learn.microsoft.com/windows/dev-home/
